packages/gitbook/src/components/Search/SearchResults.tsx

```diff
@@ -98,7 +98,7 @@ export const SearchResults = React.forwardRef(function SearchResults(
                     : searchSpaceContent(spaceId, revisionId, query));
 
                 setResults(withAsk ? withQuestionResult(fetchedResults, query) : fetchedResults);
-            }, 250);
+            }, 400);
```

Comment:
Are you sure about this? The search request already take 300-500ms so an increase debounced on top of this might degrade the perceived performance

Reply:
no not 100% as it's a delicate balance. Typing speed dictates the ~300-500ms debounce time though. Otherwise we fire off a lot of useless search queries which also causes some flashing results and pollutes analytics. 250ms is definitely too short. Let's trial it at 350 and see how it feels if this feels too drastic? Also keeping in mind that 250ms is about near instant reaction from a human perspective. Some other good news, the longer we allow the user to type before searching the faster the response will likely be. Wdyt?

Reply:
Yeah let's do that